### ISSUE TITLE
autocomplete-valid false-positive for value "xoff"

### ISSUE THREAD

**Issue by dbjorge** [dequelabs/axe-core#4877]
### Product

axe-core

### Product Version

4.10.3

### Latest Version

- [x] I have tested the issue with the latest version of the product

### Issue Description

We got the following false positive report from an auditor customer today:

#### Expectation
This input field is not relevant to SC 1.3.5 ("pet type" is not one of the enumerated types of input purposes), so it shouldn't result in an `autocomplete-valid` violation. The developer used an autocomplete attribute value of `"xoff"` as a state-like value:

```html
<input autocomplete="xoff" name="PetType" type="text">
```

#### Actual
`autocomplete-valid` violation.

#### How to Reproduce
See above

#### Additional context
#3188 #3225

**Comment by padmavemulapati** [comment]
Valdiated with the latest axe-core, develop branch code base,
not seeing failure on `auto-complete-valid` rule, when "none", "false", "true", "disabled", "enabled", "undefined", "null"
![image](https://user-images.githubusercontent.com/47409579/138279851-1b17acf9-0033-416d-a8ed-a820a8ff15ad.png)

was able to see failure earlier in this
![image](https://user-images.githubusercontent.com/47409579/138280887-64f669e9-7332-438c-a41d-a2daa310ce7f.png)
